2-1-3.py

```diff
@@ -0,0 +1,12 @@
+s = input()
+sum_k = len(s) * 60
```

Comment:
Could you give these variables a more descriptive name, please?

Reply:
yes, master!